specification/storage/resource-manager/Microsoft.Storage/preview/2020-08-01-preview/storage.json

```diff
@@ -1979,6 +1979,19 @@
       ],
       "description": "The encryption settings on the storage account."
     },
+    "ResourceAccessRule": {
+      "properties": {
```

Comment:
Should we add `action` property similar with VirtualNetworkRule and IPRule?

Reply:
That's not part of the spec.